### PR TITLE
[TD]fix jumping RTA

### PR DESCRIPTION
**Pull request by WandererFan**
Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:

- [ ] Branch rebased on latest master `git pull --rebase upstream master`
- [ ] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.19 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=34586).

---

### CODE REVIEW

File: src/Mod/TechDraw/Gui/Resources/icons/actions/techdraw-annotation.svg [hunk 1]
```diff
@@ -12,655 +12,308 @@
...
    width="64px"
    height="64px"
-   id="svg2985"
-   version="1.1"
-   inkscape:version="0.92.4 (unknown)"
-   sodipodi:docname="techdraw-annotation.svg">
+   id="svg2766"
+   sodipodi:version="0.32"
+   inkscape:version="0.92.4 (5da689c313, 2019-01-14)"
+   sodipodi:docname="Draft_Scale.svg"
```
Comment (donovaly) [inline src/Mod/TechDraw/Gui/Resources/icons/actions/techdraw-annotation.svg]: This is wrong. The name was correct before.

**Comment by WandererFan** [comment]
Thats the name the code uses.  The upper case leading character is a wiki
convention, I think.
        <file>icons/actions/techdraw-annotation.svg</file>
    sPixmap       = "actions/techdraw-annotation";

**Comment by donovaly** [comment]
> Thats the name the code uses. 

My point is that  sodipodi:docname was before correctly ""techdraw-annotation.svg"
now it was changed to "Draft_Scale.svg".
(The SVG is not for Draft, but for TD.)

**Comment by WandererFan** [comment]
Does anything in TD rely on sodipodi:docname?  Pretty sure some of the
icons are/were saved as plain SVG so don't have Inkscape extension
information anyway.

**Comment by donovaly** [comment]
> Does anything in TD rely on sodipodi:docname? 

No, but why should a correct name be changed to an incorrect one?

**Pull request merged**